app/(tabs)/transportation/status.tsx

```diff
@@ -58,17 +59,18 @@ export const StatusScreenDetails: React.FC<StatusScreenDetailsProps> = ({ school
           dropdownStyles={styles.dropdownStyles}
           boxStyles={styles.dropdownBox}
           inputStyles={styles.dropdownInput}
-          maxHeight={122}
+          maxHeight={140}
           search={false}
         />
       </View>
       <View style={styles.section}>
-        <SectionHeader text="Looking For Carpool" />
+        <SectionHeader text="Looking for Walking Buddy" />
```

Comment:
for this can you do a map/for each through each status so we can reduce redundant code